frontend/src/components/navigation/Router.tsx

```diff
@@ -18,39 +17,46 @@ import PurchaseOrderList from "../../pages/purchases/POList";
 import PasswordChangePage from "../../pages/auth/PasswordChange";
 import SalesReportPage from "../../pages/sales/SalesReport";
 import BuyBackList from "../../pages/buybacks/BuyBackList";
-import GoToLoginPageIfNotLoggedIn from "../../util/AuthCheck";
 import ShelfCalculator from "../../pages/storeplanner/ShelfCalculator";
 import VendorAdd from "../../pages/vendors/VendorAdd";
-import UserAdd from "../../pages/users/UserAdd";
 import UserList from "../../pages/users/UserList";
+import UserDetail from "../../pages/users/UserDetail";
+import { AccessType } from "../../util/UserTypes";
 
-const WithNavBar = () => {
-  return (
-    <>
-      <NavigationBar />
-      <Outlet />
-    </>
-  );
-};
+interface NavbarProps {
```

Comment:
We have NavbarProps here, and then NavigationBarProps in the Navbar file. They seem to have the exact same information too. I'm wondering if there's any way to get rid of one or the other? 

Maybe we can just get rid of this "WithNavBar" in the Router file, and just directly instantiate the <NavigationBar> in the Router. Then, we can just return an <Outlet> along with the NavBar in the Navbar file. Something similar to that, but I think we have an extra thing here that we don't really need

Reply:
Did it. Removed the unnecessary in between function.